src/webpack.config.ts

```diff
@@ -237,6 +238,25 @@ function webpackConfig(args: Partial<BuildArgs>) {
 					return [
 						{ test: /custom-element\.js/, loader: `imports-loader?widgetFactory=${args.element}` }
 					];
+				}),
+				...includeWhen(args.bundles && Object.keys(args.bundles).length, () => {
+					const loaders: any[] = [];
+
+					Object.keys(args.bundles).forEach(bundleName => {
+						(args.bundles || {})[ bundleName ].forEach(fileName => {
```

Comment:
Surprised you need to default `args.bundles` to `{}`, would have thought `Object.keys` would fail is it was undefined.